dokuwikixmlrpc.py

```diff
@@ -247,9 +247,9 @@ def append_page(self, page_id, text, summary='', minor=False):
         self._xmlrpc.dokuwiki.appendPage(page_id, text, params)
 
     @checkerr
-    def pagelist(self, namespace):
+    def pagelist(self, namespace, opts = {'depth': 0, 'hash': False, 'skipacl': False}):
```

Comment:
What do these options do? I can't really find any documentation for this.

Reply:
Here: https://www.dokuwiki.org/devel:xmlrpc#dokuwikigetpagelist

There's a link to XRef documentation on [search_allpages()](http://xref.dokuwiki.org/reference/dokuwiki/_functions/search_allpages.html)

where it says:

```
$opts['depth']   recursion level, 0 for all
$opts['hash']    do md5 sum of content?
$opts['skipacl'] list everything regardless of ACL
```